docs/home/whats-new-2-13.md

```diff
@@ -40,7 +40,7 @@ Added to the node pool grid, columns to show the current placement Strategy per
 
 <!-- RUN-9270/9274 - Interactive Time limit Fixes -->
 
-Fixed interactive time limits so that workloads that reach the timeout are now suspended/stopped. The admin can change the time limit and the timeout for new and already running workloads. Already running workloads will update and stop based on the new settings. Workloads do not reach a state of failure so that they can be resumed later.
+Fixed interactive time limits so that workloads that reach the timeout are now suspended/stopped. The admin can change the time limit and the timeout for new and already running workloads. Already running workloads will update and stop based on the new settings.
```

Comment:
```suggestion
Fixed time limits so that any workloads that reach the timeout are now suspended/stopped. The admin can change the time limit and the timeout for new and already running workloads. Already running workloads will update and stop based on the new settings.
```